Rakefile

```diff
@@ -2368,10 +2368,6 @@ def copy_sample_files
                   'base-appliances-none.xml',
                   'base-dhw-combi-tankless-outside.xml',
                   'base-dhw-indirect-outside.xml',
-                  'base-dhw-jacket-electric.xml',
-                  'base-dhw-jacket-gas.xml',
-                  'base-dhw-jacket-hpwh.xml',
-                  'base-dhw-jacket-indirect.xml',
```

Comment:
It's great to have all 4 test files in the OpenStudio-HPXML repo, but I think we can just have a single sample file in this repo since there isn't a whole lot extra being tested for the ERI calculation. How about just the 'base-dhw-jacket-gas.xml' file?

Reply:
No problem

Reply:
Thanks. It probably wasn't clear, but the main reason not to have all 4 files in this repo is to keep the CI runtime down (less simulations). The CI currently takes 45-50 mins to run and each new modeling feature w/ test files increases it 😅.

Reply:
Thanks for explanations!